modules/installation-user-infra-generate-k8s-manifest-ignition.adoc

```diff
@@ -62,22 +62,24 @@ endif::[]
 
 Because you must modify some cluster definition files and manually start the cluster machines, you must generate the Kubernetes manifest and Ignition config files that the cluster needs to make its machines.
 
+The installation configuration file is transformed into the Kubernetes manifests. Those manifests are wrapped into the Ignition configuration files, which are later used to create the cluster.
```

Comment:
"The installation configuration file transforms into the Kubernetes manifests. Manifests wrap into the Ignition config files, which are later used to create the cluster."

Reply:
Thank you for the suggestion. On review, I have incorporated some elements from your example:

"The installation configuration file transforms into the Kubernetes manifests. The manifests wrap into the Ignition configuration files, which are later used to create the cluster."